tutorials/23_MultiHop_QA_With_Agents.ipynb

```diff
@@ -0,0 +1,5680 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "h8gcxwg4sAXd"
+      },
+      "source": [
+        "# Tutorial: Multihop Question Answering with Agents\n",
+        "\n",
+        "- **Level**: Advanced\n",
+        "- **Time to complete**: 10 minutes\n",
+        "- **Nodes Used**: `Agent`, `PromptNode`, `InMemoryDocumentStore`, `FARMReader` and `ExtractiveQAPipeline`\n",
+        "- **Goal**: After completing this tutorial, you will have learned how to make use of Agents to build a multi-hop question answering system that uses an `ExtractiveQAPipeline` as a tool\n",
+        "- **Prerequisites**: An OpenAI API Key\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "collapsed": false,
+        "id": "ifCja5PasAXf"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "The Agent class makes use of a large language model to take decisions and come up with the best next course of action. You are able to provide the `Agent` with a set of `Tools` that it is able to use to reach a result. \n",
+        "\n",
+        "In this tutorial, we will let the Agent use a commonly used Haystack component, the `ExtractiveQAPipeline` to answer question. At each iteration, the agent will decide what tool to use. Based on the result returned by a tool, the Agent has two options: It will either decide to select a tool again and do another iteration, or it will decide that it has reached a conclusion and return the final answer.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jsUTxX7y5XS_"
+      },
+      "source": [
+        "## Preparing the Colab Environment\n",
+        "\n",
+        "- [Enable GPU Runtime in Colab](https://docs.haystack.deepset.ai/docs/enabling-gpu-acceleration#enabling-the-gpu-in-colab)\n",
+        "- [Set logging level to INFO](https://docs.haystack.deepset.ai/docs/log-level)"
+      ]
+    },
```

Comment:
Also now reviewing Bilge's tutorial, I can see she added a telemetry section somewhere here

Reply:
Yep, I think we should add them to all now..